src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinatorViewModel.swift

```diff
@@ -32,6 +37,8 @@ enum FileScannerError: CaseIterable {
 			return AppStrings.FileScanner.PasswordError.title
 		case .alreadyRegistered:
 			return AppStrings.FileScanner.AlreadyRegistered.title
+		default:
```

Comment:
Instead of adding a default I would prefer to add the specific case.
With the default in place a future developer would not be warned about adding an string here for a new enum case.